pyinteraph/centrality_analysis.py

```diff
@@ -61,6 +131,20 @@ def get_edge_betweenness_cent(G, **kwargs):
     reordered_dict = reorder_edge_names(centrality_dict)
     return reordered_dict
 
+def get_edge_current_flow_betweenness_cent(G, **kwargs):
+    """Returns a dictionary of edge current flowbetweenness centrality 
+    values. This is the same as calculating edge betweenness centrality 
+    using random walks instead of shortest paths.
+    """
+    
+    #G_no_isolates = remove_isolates(G)
+    centrality_dict = nxc.edge_current_flow_betweenness_centrality(\
+                                    G = G,
+                                    normalized = kwargs["normalized"],
+                                    weight = kwargs["weight"])
+    reordered_dict = reorder_edge_names(centrality_dict)
+    return reordered_dict
+
 def get_components(G, cutoff = 4):
```

Comment:
Use the function in `graph_analysis` or a simple call to the NetworkX function to get the components, and then transfer the content of this function to `get_centrality_dict`.